backend/default.nix

```diff
@@ -13,16 +13,26 @@ let
           ghcOptions = ["-Werror"];
 
           # Do not depend on GHC
```

Comment:
Perhaps this comment should be moved one line below.

Reply:
So, I moved it, but not really understand why

Reply:
Because it belongs to `dontStrip = false;`, doesn't it?